src/modules/getMeals.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable */
+import { addLike } from './userinvolve.js';
+import mealsFunction from './modalMeals.js';
+/* eslint-enable */
+const modalDetailsContent = document.querySelector('.modal-details-content');
+const recipeCloseBtn = document.getElementById('modal-close-btn');
+
+recipeCloseBtn.addEventListener('click', () => {
+  modalDetailsContent.parentElement.classList.remove('displayModal');
+});
+
+function getMealInfo(e) {
+  e.preventDefault();
+
+  if (/like-/.test(e.target.id)) {
+    const { id } = e.target;
+    const foodNum = id.match(/\d+$/)[0];
+    addLike(foodNum);
+  }
+
+  if (e.target.classList.contains('comment-btn')) {
+    const mealItem = e.target.parentElement.parentElement;
+    fetch(
+      `https://www.themealdb.com/api/json/v1/1/lookup.php?i=${mealItem.dataset.id}`,
+    )
+      .then((response) => response.json())
+      .then((data) => mealsFunction(data.meals));
+  }
```

Comment:
- [ ] As suggested by the previous code reviewer. I kindly suggest that you make use of async and await rather than promises (then) with the fetch method to send and receive data.


## EXAMPLE
 ```
 async function getMealById(mealId) {
      const response = await fetch(
        `https://www.themealdb.com/api/json/v1/1/lookup.php?i=${mealItem.dataset.id}`,
      );
  
      const data = await response.json();
      const meal = data.meals[0];
      return meal;
    }
```